Automattic/jetpack
# Subs: Don't load block when offline

Fixes #26178

#### Changes proposed in this Pull Request:

* The subscriptions block should not register when in Offline mode. This prevents that.

#### Other information:


#### Jetpack product discussion


n/a

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* See #26178
* Before patch: when loading Block Editor, see the error when loading localhost.
* After patch, no error.
